example/basic/index.js

```diff
@@ -1,3 +1,4 @@
+'use strict'
```

Comment:
yeah, think that's best - afaik I haven't added it anywhere, not sure if
someone
else snuck it in tho

On Sat, Oct 22, 2016 at 6:29 PM Arve Knudsen <notifications@github.com>
wrote:

> In example/basic/index.js
> <https://github.com/yoshuawuyts/bankai/pull/84#discussion_r84580354>:
>
> > @@ -1,3 +1,4 @@
> > +'use strict'
>
> @yoshuawuyts <https://github.com/yoshuawuyts> Cool, we should just remove
> it consistently from all source files then? I was actually wondering what
> we should do on a project wide basis, since the linter doesn't care.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/yoshuawuyts/bankai/pull/84/files/b979e25e0e04c0fcfbad812df90711c9dca5b9b2#r84580354>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWletvImeDldvg3kq1uouRWFm4-orH-ks5q2jnYgaJpZM4Kd2LR>
> .
>